src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/RecapitiPGPagoPaTest.java

```diff
@@ -87,4 +88,29 @@ public void nellaPaginaITuoiRecapitiSiControllaCheCiSiaGiaUnaPec() {
             backgroundTest.aggiungiPECPG();
         }
     }
+
+    @And("Si clicca su elimina email")
+    public void siCliccaSuEliminaEmail() {
+        ITuoiRecapitiPage iTuoiRecapitiPage = new ITuoiRecapitiPage(this.driver);
+
+        iTuoiRecapitiPage.eliminaEmailEsistente();
+    }
+
+    @And("Si annulla eliminazione email")
+    public void siAnnullaEliminazioneEmail() {
+        recapitiDestinatarioPage.checkButtonAnnullaEliminazioneInPopUp();
+        recapitiDestinatarioPage.clickButtonAnnullaEliminazioneInPopUp();
+    }
+
+    @And("Si conferma eliminazione nel pop up")
+    public void siConfermaEliminazioneNelPopUp() {
+        if (recapitiDestinatarioPage.waitLoadPopUpElimina().equalsIgnoreCase("Rimuovi e-mail")) {
+            recapitiDestinatarioPage.confermaButtonEliminaClick();
+        }
+    }
+
+    @And("Si controlla presenza email precedentemente salvata {string}")
+    public void siControllaPresenzaEmailPrecedentementeSalvata(String email) {
+        recapitiDestinatarioPage.checkEmailPrecedentementeSalvata(email);
+    }
```

Comment:
Is it really necessary to have single steps instead of one that performs everything outlined in the feature step?

Reply:
it might be useful in the future